client/src/components/SignUpPage.vue

```diff
@@ -26,17 +27,22 @@
 
 <script>
 import Navbar from "./Navbar";
+import {AUTH_REQUEST} from "../store/auth";
 
 export default {
   name: "SignUpPage",
   components: { Navbar },
   data() {
-    return {};
+    return {
+      email : "",
+      password : "",
+      confirmpassword : "",
```

Comment:
trailing comma

Reply:
+ in js code You can use sinlge `'`, leave double ones to html

Reply:
Both practices have been set up by @fwojtowicz in the initial commit. Should we follow with trailing commas or not? Singlequotas or doublequotas?

Reply:
is this practice or linting preset? If second then too bad, still can be changed and linted afterwards. If its practice its simply not needed. On the overall Its poorly linted, its good practice to run lint before commiting 😉

Reply:
Quotation marks were fixed (I had to reconfigure linter) and code was pushed to master branch without `"`. And when it comes to trailing commas, "," between `components` and `data` is required, it is not a trailing comma